br/pkg/lightning/restore/meta_manager.go

```diff
@@ -78,6 +78,7 @@ func (b *dbMetaMgrBuilder) TableMetaMgr(tr *TableRestore) tableMetaMgr {
 type tableMetaMgr interface {
 	InitTableMeta(ctx context.Context) error
 	AllocTableRowIDs(ctx context.Context, rawRowIDMax int64) (*verify.KVChecksum, int64, error)
+	ReallocTableRowIDs(ctx context.Context, newRowIDCount int64) (int64, int64, error)
```

Comment:
Please add more comments for this method, since `noopTableMetaMgr` doesn't really implement it.

Reply:
[4b5b415](https://github.com/pingcap/tidb/pull/34288/commits/4b5b4155917d93ec5564b870412ac07075fb438d)